api/client/src/main/java/org/projectnessie/client/http/impl/jdk8/UrlConnectionClient.java

```diff
@@ -54,4 +54,9 @@ public HttpRequest newRequest() {
   public URI getBaseUri() {
     return config.getBaseUri();
   }
+
+  @Override
```

Comment:
If it makes sense, feel free to make `HttpClient` an abstract base class

Reply:
For now it's just one method that needs implementation in each concrete impl: `close()`, so I think it's not (yet) justified to make this change.

Reply:
Okay for me